src/validators/typed_dict.rs

```diff
@@ -143,8 +123,7 @@ impl BuildValidator for TypedDictValidator {
 
         Ok(Self {
             fields,
-            check_extra,
-            forbid_extra,
+            extra_behavior,
```

Comment:
might be easier to keep the old `check_extra`, `forbid_extra` behaviour, and avoid rewriting below?

Reply:
I am now using the `extra_behavior` enum everywhere. It creates consistency between all of the validators, I think it's better than using a boolean here.